src/structures/TextChannel.js

```diff
@@ -53,6 +53,7 @@ class TextChannel extends GuildChannel {
 
     /**
      * The ratelimit per user for this channel in seconds
+     * <warn>It is not currently possible to set a rate limit per user on a `NewsChannel`.</warn>
```

Comment:
I'm unsure about this warn being in TextChannel's docs, though.

Reply:
Well it will default to 0 but I figured it’d make sense to say it doesn’t exist. Want me to remove it?

Reply:
It will actually be set to undefined for some wild reason, so this warn might justified.
https://github.com/discordjs/discord.js/blob/2eac84296b448907213680690ec766bb5fbe5990/src/structures/NewsChannel.js#L15

Reply:
Well perfect then. Is it ready to be merged?